recipes/openmpi/meta.yaml

```diff
@@ -0,0 +1,73 @@
+{% set version = "2.0.0" %}
+{% set major = ".".join(version.split(".")[:2]) %}
+
+package:
+    name: openmpi
+    version: {{ version }}
+
+source:
+    fn: openmpi-{{ version }}.tar.bz2
+    url: https://www.open-mpi.org/software/ompi/v{{ major }}/downloads/openmpi-{{ version }}.tar.bz2
+    sha1: a34f07a0e94af4ccd8080d74827befd61824c6c3
+
+build:
+    number: 0
+    skip: True  # [win]
+    binary_relocation: true
+    detect_binary_files_with_prefix: true
+
+requirements:
+    build:
+        - perl
+        - gcc  # [linux]
+    run:
+        - libgcc  # [linux]
+
+test:
+    files:
+        - tests/helloworld.c
+        - tests/helloworld.cxx
+    commands:
+        - command -v ompi_info
+        - ompi_info
+
+        - command -v mpirun
+        - mpirun --allow-run-as-root --help
+
+        - command -v mpicc
+        - mpicc --allow-run-as-root -show
+
+        - command -v mpicxx
+        - mpicxx --allow-run-as-root -show
+
+        - command -v mpif90  # [linux]
+        - mpif90 --allow-run-as-root -show  # [linux]
+
+        - export DYLD_LIBRARY_PATH=$PREFIX/lib  # [osx]
+        - command -v mpiexec
+        - mpicc $RECIPE_DIR/tests/helloworld.c -o helloworld_c
+        - mpiexec -mca plm isolated --allow-run-as-root -n 4 helloworld_c
```

Comment:
I had to add the `-mca plm isolated` to be able to make this work in the `docker` image.